pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -99,6 +99,12 @@ func validateServiceBrokerSpec(spec *sc.ServiceBrokerSpec, fldPath *field.Path)
 			)
 		}
 
+		if spec.RelistBehavior == sc.ServiceBrokerRelistBehaviorDuration && spec.RelistDuration == nil {
```

Comment:
Question: do you think it is valid for duration to be set if the behavior is manual?  It might be.  If you switch from duration to manual temporarily, for example, you might not expect to have to unset the duration.  Consider what the 'least surprise' is here.

Reply:
Also, I don't see unit tests for the validations

Reply:
> do you think it is valid for duration to be set if the behavior is manual? It might be. If you switch from duration to manual temporarily, for example, you might not expect to have to unset the duration.

Agreed. I would be surprised if I had to explicitly unset duration, and instead would expect the duration to simply be ignored.